akka-stream/src/main/boilerplate/akka/stream/FanInShape.scala.template

```diff
@@ -7,18 +7,24 @@ import scala.collection.immutable
 import scala.annotation.varargs
 
 object FanInShape {
-  sealed trait Init[+O]
-  case class Name(name: String) extends Init[Nothing]
-  case class Ports[O](out: Outlet[O], ins: immutable.Seq[Inlet[_]]) extends Init[O]
+  sealed trait Init[+O] {
```

Comment:
The number of objects created and retained is the same as before, but I don't mind either way.